test/benchmark_min_time_flag_time_test.cc

```diff
@@ -0,0 +1,90 @@
+#include <cassert>
+#include <climits>
+#include <cmath>
+#include <cstdlib>
+#include <cstring>
+#include <iostream>
+#include <string>
+#include <vector>
+
+#include "benchmark/benchmark.h"
+
+// Tests that we can specify the min time with
+// --benchmark_min_time=<NUM> (no suffix needed) OR
+// --benchmark_min_time=<NUM>s
```

Comment:
Is there a negative test for `1ms` / `1z` / `1hs`?

Reply:
Yes - just added a set of death tests for these